### PR TITLE
Update code-climate.md to include Python instructions

### PR DESCRIPTION
**Pull request by efueger**

### CODE REVIEW

File: user/code-climate.md [hunk 1]
```diff
@@ -85,6 +85,20 @@ data to Code Climate requires adding the following to your .travis.yml:
...
 
+### Python
+
+Python projects can use [Code Climate's Python
+reporter](https://github.com/codeclimate/python-test-reporter) to collect code coverage
+data.
+
+To set it up for your project, follow the instructions in the
+[README](https://github.com/codeclimate/python-test-reporter#codeclimate-test-reporter).
+
```
Comment (plaindocs) [inline user/code-climate.md]: We could delete "+Generally, you'll need to:" entirely? and just follow on from "To set it up........:"

File: user/code-climate.md [hunk 2]
```diff
@@ -85,6 +85,20 @@ data to Code Climate requires adding the following to your .travis.yml:
...
+Generally, you'll need to:
+1. specify the repository token in your .travis.yml OR configure the repo token in the environment through your CI settings
+2. generate a test report with coverage.py: https://coverage.readthedocs.io
+3. run the `codeclimate-test-reporter` command (either with the token passed as an argument, or as a stored env variable)
+
```
Comment (plaindocs) [inline user/code-climate.md]: Could you punctuate the list items, mark `.travis.yml` `coverage.py` as code and link the URL? Thanks!

**Comment by plaindocs** [comment]
Thanks! If you could add the couple of fixes inline I'll merge this in. :-)

**Comment by efueger** [comment]
Thanks for the recommendations! 
- I've updated the file with the changes (plus some reconfigurations). 
- Not sure if making changes inline meant submitting a new commit to the PR, so let me know if you needed me to do that some other way.

**Comment by plaindocs** [comment]
Awesome!

**Pull request merged**